### DISCUSSION TITLE
Windows 11 24H2 install the latest 9968 in the DTS license are shown trial and purchase

### DISCUSSION THREAD

**Discussion by bluecloud2021** [AAFOptimus/AAFFamilyDCHAudio#3] [Q&A]
Windows 11 24H2 install the latest 9968 in the DTS license are shown trial and purchase, no authorization successful, please fix

**Comment by AAFOptimus** [comment]
Make sure your DTS Sound Unbound app is automatically updated to the latest version available in the Microsoft Store. If yes, revert it to the version available in this package.

![{2D96CD77-9E4B-42B4-A2BA-11BE47269CBA}](https://github.com/user-attachments/assets/659d8a46-5a14-4823-84bd-f4529077daa3)

![{D1C38D5A-7496-4562-816B-97E714C7DE04}](https://github.com/user-attachments/assets/afa5e955-7931-45e2-b9f7-65fee7ce0628)

![{764F4C74-F4B6-4129-9B46-6000E6A9AADE}](https://github.com/user-attachments/assets/dfc5abad-fc15-4977-89a6-9537fb314d65)

![{FF2849D5-B1D9-40FD-8371-69E84F1A3953}](https://github.com/user-attachments/assets/34219f05-124a-49cb-82d6-3021d6b7ca03)

![{1A110B85-F053-41D8-9687-310941A9EED6}](https://github.com/user-attachments/assets/1d4c6d54-b393-4446-a23a-4ddae75ff08d)
